homeassistant/components/growatt_server/__init__.py

```diff
@@ -1 +1,33 @@
 """The Growatt server PV inverter sensor integration."""
+from homeassistant import config_entries
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DOMAIN
+
+
+async def async_setup(hass, config):
+    """Set up this integration."""
+    if DOMAIN in config:
+        for device in config[DOMAIN]:
+            hass.async_create_task(
+                hass.config_entries.flow.async_init(
+                    DOMAIN,
+                    context={"source": config_entries.SOURCE_IMPORT},
+                    data=device,
+                )
+            )
```

Comment:
Remove this. There's no existing configuration yaml under the integration domain key.
```suggestion
```